.gitignore

```diff
@@ -21,4 +21,6 @@ obj/
 _ReSharper*/
 *sln.dotsettings
 /packages/
-!packages/repositories.config
\ No newline at end of file
+!packages/repositories.config
+/.vs/SharpBucket/v15/Server/sqlite3
+*.nupkg
```

Comment:
Why should we exclude *.nupkg explicitly ?
I'm convinced that it's a type of file that should not be pushed, but it's also a type of file that is not expected to appear in a not ignored path inside the repository...